### PR TITLE
Minor documentation UI improvements

### PR DESCRIPTION
**Pull request by aniketkatkar97**
- [x] Changed the favicon.
- [x] Updated the contact us mail id.

Screenshot 2023-05-19 at 9 52 16 AM

- [x] Changed the skeleton loader layout for a better view.

### CODE REVIEW

File: constants/SkeletonLoader.constants.ts [hunk 1]
```diff
@@ -14,4 +14,7 @@ export const SKELETON_PARAGRAPH_WIDTHS = [
   undefined,
   undefined,
   SkeletonWidth.SMALL,
+  SkeletonWidth.LARGE,
+  SkeletonWidth.LARGE,
+  SkeletonWidth.LARGE,
```
Comment (chirag-madlani) [inline constants/SkeletonLoader.constants.ts:19]: Is it intentional?

File: components/Footer/Footer.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default function Footer() {
             Home
           </Link>
           <Link
-            href="mailto:hello@getcollate.io?subject=Contact%20from%20documentation%20"
+            href="mailto:sales@getcollate.io?subject=Contact%20from%20documentation%20"
```
Comment (Sachin-chaurasiya) [inline components/Footer/Footer.tsx:22]: Move this link to constant.

File: pages/_app.tsx [hunk 3]
```diff
@@ -26,8 +26,7 @@ export default function MyApp({ Component, pageProps }: AppProps<MyAppProps>) {
     <>
       <Head>
         <title>{TITLE}</title>
-        <link rel="icon" href="/favicon.svg" />
-        <link rel="alternate icon" href="/favicon32.ico" />
```
Comment (Sachin-chaurasiya) [inline pages/_app.tsx:30]: Why we are removing the alternate icon?

File: constants/SkeletonLoader.constants.ts (hunk 1 above)
Comment (aniketkatkar97) [reply constants/SkeletonLoader.constants.ts:19]: Yes

File: pages/_app.tsx (hunk 3 above)
Comment (aniketkatkar97) [reply pages/_app.tsx:30]: Thanks. I'll revert that

**Review by Sachin-chaurasiya — APPROVED** [review]
Looks Good 👍

**Pull request merged**